### PR TITLE
Modifying the Dockerfile to add the needed pip3 update

### PR DESCRIPTION
**Pull request by bmos299**
### Description
Modifying the Dockerfile so images can be built
### Fixes
I just had to dd changes for the pip3 update

### CODE REVIEW

File: containers/kraken.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ spec:
         - name: kraken
           securityContext:
             privileged: true
-          image: quay.io/openshift-scale/kraken
+          image: bmos299/kraken-debug
```
Comment (chaitanyaenr) [inline containers/kraken.yml:21]: We might want to skip updating the image, an image build will be triggered once we merge this PR: https://quay.io/repository/openshift-scale/kraken.

**Comment by chaitanyaenr** [comment]
Fixes #87

File: containers/kraken.yml (hunk 1 above)
Comment (bmos299) [reply containers/kraken.yml:21]: absolutely...i didnt mean to commit this.

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Pull request merged**